adcircpy/forcing/winds/best_track.py

```diff
@@ -280,6 +322,16 @@ def make_request():
 
             self.__atcf = io.BytesIO(response.read())
 
+    @property
+    def track_length(self) -> float:
```

Comment:
@JaimeCalzadaNOAA is there a way to cache the result of this so it isn't calculated over again in the case I call the attribute?

Reply:
I just implemented one of the multiple ways to do it. There are many other ways, but this is the one you will probably see most often used throughout my codes.